index.d.ts

```diff
@@ -14,11 +19,11 @@ export type VerificationPublicKey = {
 };
 
 interface VerifyRequestInterface {
-  (rawBody: string, signature: string, key: string): Promise<boolean>;
+  (rawBody: string, signature: string, key: string): boolean;
```

Comment:
The `verifyRequset` method is async, we need to keep the `Promise<...>`
```suggestion
  (rawBody: string, signature: string, key: string): Promise<boolean>;
```

Reply:
Meant to follow up on this: I originally walked through the block of code and, originally saw that the operations were all synchronous (assertions; createVerify; update; verify). However, seeing the prepended `async` on the `.verifyRequest` method helped me see why we should be returning a `Promise<boolean>` there -- thanks for the correction @gr2m!